src/Loader.php

```diff
@@ -240,52 +238,6 @@ public static function load_features() {
 		}
 	}
 
-	/**
-	 * Adds the Features section to the advanced tab of WooCommerce Settings
-	 *
-	 * @param array $sections Sections.
-	 * @return array
-	 */
-	public static function add_features_section( $sections ) {
-		$sections['features'] = __( 'Features', 'woocommerce-admin' );
-		return $sections;
-	}
-
-	/**
-	 * Adds the Features settings.
-	 *
-	 * @param array  $settings Settings.
-	 * @param string $current_section Current section slug.
-	 * @return array
-	 */
-	public static function add_features_settings( $settings, $current_section ) {
-		if ( 'features' !== $current_section ) {
-			return $settings;
-		}
-
-		return apply_filters(
-			'woocommerce_settings_features',
```

Comment:
Instead of removing this entirely, could we use a filter to add the feature setting in from Navigation?  That way when we turn it on, we'll have this option available again.

Reply:
This code is still on `main` branch. I will create an issue to handle 1.9 Nav requirements and think about how to handle this code and its migration to wc-calypso-bridge